### PR TITLE
GPII-4272: Update instructions for deploying and performing QA on a new universal container.

### PR DESCRIPTION
**Pull request by the-t-in-rtf**
See [https://issues.gpii.net/browse/GPII-4272](https://issues.gpii.net/browse/GPII-4272) for details.

IMO this should be reviewed and merged after [Joseph's pull to add the ability to more easialy run the production tests](https://github.com/gpii-ops/gpii-infra/pull/347)

### CODE REVIEW

**Comment by natarajaya** [comment]
@the-t-in-rtf This is very helpful, thanks!

LGTM

**Comment by the-t-in-rtf** [comment]
I simplified the instructions regarding installing credentials based on later testing and my conversations with @javihernandez.

File: TESTING.md [hunk 1]
```diff
@@ -58,13 +54,41 @@ $ node_modules\.bin\electron . configs app.cloud
...
+You can find the Morphic Installers in our [shared Google Drive](https://drive.google.com/drive/u/1/folders/1nzXsW83qRejup3D_yVYO9t5i4lKN_p5G).  Please note, this link will only work if you are logged in using a Google account that has permission.
+
+Run the installer you want to test.  When installation is finished, you will need to manually configure the client to use your dev cloud.
+
+1. Edit `C:\Program Files (x86)\Morphic\windows\service.json5`
+1. Update the `GPII_CLOUD_URL` setting to point to your dev cloud, i.e. `https://flowmanager.<username>.dev.gcp.gpii.net`
+1. To avoid errors with the self-signed certificates used in a dev cloud, add a new `NODE_TLS_REJECT_UNAUTHORIZED` and set it to `0`. 
+1. Restart, either by:
+   1. Restarting windows
+   1. Opening PowerShell as an administrator and running `restart-service "morphic service"`
```
Comment (natarajaya) [inline TESTING.md]: @the-t-in-rtf While working on GPII-4159, I noticed that this command always crashes with
```
PS C:\Users\vagrant> restart-service "morphic service"
restart-service : Service 'morphic service (morphic-service)' cannot be stopped due to the following error: Cannot
open morphic-service service on computer '.'.
At line:1 char:1
+ restart-service "morphic service"
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (System.ServiceProcess.ServiceController:ServiceController) [Restart-Service
   ], ServiceCommandException
    + FullyQualifiedErrorId : CouldNotStopService,Microsoft.PowerShell.Commands.RestartServiceCommand
```

Comment (the-t-in-rtf) [reply TESTING.md]: The command works with the released installer, as long as you are running PowerShell as an administrator.  If you are running PowerShell as a normal user, you will see exactly that message.  Can you confirm whether you're running PowerShell as an admin?

Comment (natarajaya) [reply TESTING.md]: @the-t-in-rtf Looks like default vagrant user lacks admin privileges, running as admin solves this indeed, thanks!

**Comment by stepanstipl** [comment]
LGTM

**Pull request merged**